ote/src/clj/ote/services/admin.clj

```diff
@@ -73,6 +76,36 @@
          search-params
          {:specql.core/order-by ::t-service/operator-name})))
 
+(defn distinct-by [f coll]
+  (let [groups (group-by f coll)]
+    (map #(first (groups %)) (distinct (map f coll)))))
+
+(defn- business-id-report [db user query]
+  (let [services (if
```

Comment:
Use `when` instead of `(if condition value nil)` pattern

Reply:
Changed